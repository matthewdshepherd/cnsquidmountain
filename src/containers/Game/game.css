.game--body {
    display: grid;
    grid-template-columns: 1fr 1fr 1fr;
    grid-template-areas: 'header header header' 'main main aside';
    height: 100%;
    /* width: 100vw; */
    color: #FFF;
    width: 100%;
}

.game__header {
 grid-area: header;
 display: flex;
 justify-content: space-between;
 height: 10vh;
}

main {
 grid-area: main;
 padding: 0 20px 20px 20px;
 height: 90vh;
}

aside {
 grid-area: aside;
}

.game--name--startover {
    display: flex;
    align-items: center;
}

.game--logo--title__div {
    display: flex;
    align-items: center;
    width: 70%;
}

.game--logo {
    height: 100%;
    width: auto;
}
  
.game--title {
    color: black;
    font-size: 70px;
    margin: 0;
}

.game--name--startover {
    align-items: flex-start;

}

.game--name__h3--text {
    margin: 0;
    color: orange;
    text-shadow: 2px 2px black;
    font-size: 30px;
}

.game--startover--button {
    
}