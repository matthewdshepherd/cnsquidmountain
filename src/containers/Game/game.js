import React from 'react'
import { bindActionCreators } from 'redux';
import { connect } from 'react-redux'
import './round.css'

export class Game extends React.Component {
    constructor() {
        super()
        this.state = {
            completedWords = [] // id's of current words will live here
        }
    }



    render() {
        return (
            <div>

            </div>
        )
    }
}

export default Game