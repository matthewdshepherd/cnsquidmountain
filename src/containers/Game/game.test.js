import React from 'react'
import { shallow } from 'enzyme'
import { isTSAnyKeyword, exportAllDeclaration } from '@babel/types'
import { Game, matStateToProps } from './game'

describe('Game', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Game />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
