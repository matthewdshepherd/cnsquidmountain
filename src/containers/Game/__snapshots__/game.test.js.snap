// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game should match the snapshot 1`] = `
<div
  className="game--body"
>
  <header>
    <div
      className="game--logo--title__div"
    >
      <img
        alt="Squid Mountain Logo"
        className="game--logo"
        src="squid_mountain_logo.png"
      />
      <h1
        className="game--title"
      >
        Squid Mountain
      </h1>
    </div>
    <div
      className="game--name--startover"
    >
      <h3
        className="game--name__h3--text"
      />
    </div>
  </header>
  <Connect(Round) />
</div>
`;
